# Type
uint8 R2 = 21
uint8 SO2 = 22
uint8 SE2 = 23
uint8 R3 = 31
uint8 SO3 = 32
uint8 SE3 = 33

int32 type

# Grids
float64[] values

# Number of division
int32[] divide_nums

# Min/max position of samples
float64[] min
float64[] max
